doc/integrator/make.rst

```diff
@@ -79,24 +79,25 @@ and removes it on the `clean` rule:
 
```

Comment:
Does this section really still use mapcache.xml as an example as said in the above sentence? All the following commands are about WMTSCapabilities.xml...